code/modules/surgery/organs/tongue.dm

```diff
@@ -92,6 +92,54 @@
 		message = lizard_eckS.Replace(message, "ECKS$1")
 	speech_args[SPEECH_MESSAGE] = message
 
+/obj/item/organ/tongue/lizard/silver
+	name = "silver tongue"
+	desc = "A genetic branch of the high society Silver Scales that gives them their silverizing properties. To them, it is everything, and society traitors have their tongue forcibly revoked. Oddly enough, it itself is just blue."
+	icon_state = "silvertongue"
+	actions_types = list(/datum/action/item_action/organ_action/statue)
+
+/datum/action/item_action/organ_action/statue
+	name = "Become Statue"
+	desc = "Become an elegant silver statue. It's durability and yours are directly tied together, so make sure you're careful."
+	var/ability_cooldown = 0
+	var/obj/structure/statue/custom/statue
+
+/datum/action/item_action/organ_action/statue/New(Target)
+	. = ..()
+	statue = new(null)
+
+/datum/action/item_action/organ_action/statue/Trigger()
+	. = ..()
+	if(!iscarbon(owner))
+		to_chat(owner, "<span class='warning'>Your body rejects the powers of the tongue!</span>")
+		return
+	var/mob/living/carbon/becoming_statue = owner
+	if(ability_cooldown >= world.time)
+		to_chat(becoming_statue, "<span class='warning'>You just used the ability, wait a little bit!</span>")
+		return
+	var/is_statue = becoming_statue.loc == statue
+	to_chat(becoming_statue, "<span class='notice'>You begin to [is_statue ? "break free from the statue" : "make a glorious pose as you become a statue"]!</span>")
+	if(!do_after(becoming_statue, 5 SECONDS, target = statue))
+		to_chat(becoming_statue, "<span class='warning'>Your transformation is interrupted!</span>")
+		ability_cooldown = world.time + 3 SECONDS
+		return
+	ability_cooldown = world.time + 10 SECONDS
+
+	if(statue.name == initial(statue.name)) //statue has not been set up
+		statue.name = "statue of [becoming_statue.real_name]"
+		statue.desc = "statue depicting [becoming_statue.real_name]"
+		statue.set_custom_materials(list(/datum/material/silver=MINERAL_MATERIAL_AMOUNT*5))
```

Comment:
Yeah but silver doesn't DO that right

Reply:
lowkey, this ability still isn't gonna be used (more than once to test it out), even if the statue could reflect lasers- it's much worse than just entering a locker and closing it on yourself, due to the long wind-up times the ability demands before use and the long cooldowns it has.

now, if the statue COULD reflect lasers (but NOT bullets), turning into a statue and back was instant (without a do_after()), AND this ability had no cooldown, THEN it could be very interesting, especially combined with the inability of the silver lizards to use guns- it'd effectively be a nerfed version of sleeping carp's projectile deflection (it'd only work on lasers and beams, not all projectiles, and it'd keep you from moving (or attacking) while it's active), giving the silver lizards a unique playstyle in comparison to the other types of pirates.

Reply:
also, check to see what happens if you try to sell these statues while someone is still inside of them.

Reply:
> Yeah but silver doesn't DO that right

you could change the fluff and make it turn you into a glass golem (and back) if you wanted it to be much easier to code into the game/recognize that hey, maybe you shouldn't shoot lasers at this thing

Reply:
i mean maybe its polished silver
if it reflects light it reflects lasers!!
also yeah the main reason is because 5 egun shots will insta-kill-no-revive you from full health in statue mode, and in statue mode you're kinda a sitting duck. I'd say either drastically reduce the cooldown (so it can be used to quickly enter in stealth situations) or give it some kind of armor buff like reflecting lasers (to be useful situationally in combat while still requiring quick thinking/decision making)

Reply:
Will lower CD